signedauth/example/mgr.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"crypto/md5"
+	"crypto/sha1"
+	"encoding/hex"
+	"errors"
+	"github.com/ChristopherRabotin/gin-contrib/signedauth"
+	"hash"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+// StrictSHA1Manager is an example definition of an AuthKeyManager struct.
+type StrictSHA1Manager struct {
+	prefix string
+	secret string
+	key    string
+	value  interface{}
```

Comment:
@manucorporat , this is an example of a struct which implements the `SignedAuthManager` interface. Yes, in order to be exported, `preifx`, `secret`, `key` and `value` should start with an uppercase letter. However, in this example, there is no need for these variables to be exported, and therefore they aren't.